README.md

```diff
@@ -13,6 +13,11 @@ Pass a comma-separated list of modules to run a subset of the plugins:
 $ botbot-shell brain,images
 ```
 
+## Docs
+[![Read the Docs](https://img.shields.io/readthedocs/pip.svg)](https://brainzbot.readthedocs.io/)
+
+BrainzBot's documentation, including information on how to create new plugins, is available on [ReadTheDocs](https://brainzbot.readthedocs.io).
```

Comment:
The official spelling seems to be "Read the Docs".

Reply:
Sorry, amended!